posts/2023-07-15-riviera-dev.adoc

```diff
@@ -0,0 +1,162 @@
+= Riviera DEV 2023
+VincentBrule
+v1.0, 2023-07-15
+:title: Riviera DEV 2023
+:imagesdir: ../media/2023-07-15-riviera-dev
+:lang: en
+:tags: [conference, nice]
+
+== Riviera DEV ?
+After a great edition on a lake, https://rivieradev.fr/[Riviera DEV] is back in the Skema school from 10th to 12th July in Nice Sophia Antipolis.
+
+The conference takes place over three days, with several different talk formats.
+Riviera DEV's signature format is "We're not just coders" workshops.
+This format allows you to present a subject that is not purely technical, such as a passion.
+I'll be presenting one of the talks in this format later in the article.
+There were also some sports sessions for those who were brave enough in the sweltering heat!
+You can find the detailed programme https://rivieradev.fr/program[here].
+
+.Riviera Dev 2023
+image::talks.png[Riviera Dev's different talk formats, width = 700]
+
+=== The organisation
+Before I present my favourites from this year, I'd like to thank all the volunteers who make this conference possible.
+
+.Many thanks to the organizers
+image::end.jpg[Many thanks to the organizers and see you next year, width = 700]
+
+Apart from organizing this great conference, they are also providing financial support to several players in the Nicoise region.
+For example, https://lesmarabouteurs.fr/[a robotics club based in Antibes].
+Thanks to Riviera DEV, this club was able to participate in an international competition this year.
+They also bought PCs for teachers and trained them in the https://scratch.mit.edu/[Scratch] language.
+A big bravo to them!
+
+I'm now going to summarise three talks that I really enjoyed, and it was a tough choice to choose only three!
+If you'd like more information than my summary, all the talks were recorded and will be published shortly.
+
+== "Construire soi-même son clavier mécanique, idée bizarre ou idée de génie" (Translation: "Building your own mechanical keyboard: a bizarre idea or a stroke of genius?) - https://rivieradev.fr/orateur/1029[Benjamin Legrand]
+
+This talk was of particular interest to me, as I've been trying for several months to make my workspace as ergonomic as possible following a wrist pain.
+After presenting the history of keyboards and how they have evolved over time, we discovered the problems with current layouts.
+
+The first point that is not intuitive is the staggered layout of the keys, which comes from the first typewriters.
+In fact, an orthogonal layout is more ergonomic because you just have to make a vertical movement to change lines without having to twist your wrists.
+
+.Staggered vs Orthogonal
+image::orthogonal.png[Staggered vs Orthogonal, width = 800]
+
+The second point is the angle of the wrists to the keyboard.
+Ideally, our wrists should be as shown in the photo below.
+I dare you to be in this position with a conventional keyboard, it's really not pleasant.
```

Comment:
To avoid the comma splice here, I would recommend a period or exclamation point instead of the comma.